missions/2PzD_Template_v3_0.VR/customization/loadouts/Russian/gearRus41.sqf

```diff
@@ -85,10 +84,10 @@
     R41_PC = ["R41_PC", {
         params ["_unit"];
 
-        [Rus_Uni_PC] call Olsen_FW_FNC_AddItemRandom;
+        [Rus_Uni41_PC] call Olsen_FW_FNC_AddItemRandom;
         [Rus_Vest_PC] call Olsen_FW_FNC_AddItem;
         [Rus_Hat_PC] call Olsen_FW_FNC_AddItem;
-        [GEN_Face_r] call Olsen_FW_FNC_AddItemRandom;
+        Rus_Face;
```

Comment:
Is this correct? Not sure how this is called

Reply:
The definition is in the `gearDefRusRegular` and `gearDefRusWinter` files. Did this to make it easier to change, otherwise the required facewear would need to be placed in every loadout definition. Winter loadouts get a balaclava and gloves, for the regular loadouts we had to add something as well to make it work so chose a watch.

Reply:
Cheers for explaning it to me